controllers/WikiController.php

```diff
@@ -169,6 +169,7 @@ public function actionCreate()
         }
 
         $model = new Wiki();
+        $model->loadDefaultValues();
```

Comment:
In table `wiki` property `status` have default value as `3`. When creating a `wiki` the `status` is not set in the code, so in method afterSave `$changedAttributes ['status']` does not exist. But it should, because the `status` is set.

Reply:
This does not allow me to track the `status` change in `afterSave` when creating a `wiki`.

Reply:
OK.